city_metrix/layers/world_pop.py

```diff
@@ -1,3 +1,6 @@
+from dask.diagnostics import ProgressBar
+import xarray as xr
+import xee
 import ee
 
 from .layer import Layer, get_image_collection
```

Comment:
Should this be `from .layer import Layer, get_utm_zone_epsg, get_image_collection`?

Reply:
fixed